concept/thing/impl/AttributeImpl.java

```diff
@@ -88,7 +88,9 @@ public FunctionalIterator<ThingImpl> getOwners() {
 
     @Override
     public FunctionalIterator<ThingImpl> getOwners(ThingType ownerType) {
```

Comment:
I think this method is doing what it should be doing: getting owners of this attribute from the owner type, including its subtypes. It shouldn't traverse up the types.